tests/base.py

```diff
@@ -374,6 +377,18 @@ def func_wrapper(self):
     return func_wrapper
 
 
+def change_admin_user_role_to_super_admin(func):
```

Comment:
Why do you need this function? There is a similar one in this file.

Reply:
@rajeman this wrapper is helping me to find if the role of the user who is trying to change another user to super admin is himself a super_Admin. The other function creates the super Admin, and if I use it, the super admin will be duplicated hence the test database will complain.